datasrc/network.py

```diff
@@ -403,11 +403,125 @@
 		NetStringStrict("m_Reason"),
 		NetBool("m_Force"),
 	]),
-	
+
 	### ModAPI
-	
+
 	NetMessage("ModAPI_Sv_Broadcast", [
 		NetStringStrict("m_pMessage"),
 	]),
-	
+
+
+	### NetGUI
+	## Server
+	NetMessage("ModAPI_Sv_GuiRemoveElement", [
+		NetIntAny("m_Type"),
+		NetIntAny("m_ID"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiRequestData", [
+		NetIntAny("m_ID"),
+		NetIntAny("m_Type"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiUIRect", [
+		NetIntAny("m_ID"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetArray(NetIntRange("m_Color", 0, 100), 4),
+		NetIntRange("m_Corner", 0, 255),
+		NetIntRange("m_RoundingX10", 0, 500),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiLabel", [
+		NetIntAny("m_ID"),
+		NetString("m_Text"),
+		NetArray(NetIntRange("m_Dimension", 0, 100), 4),
+		NetArray(NetIntRange("m_Color", 0, 100), 4),
+		NetIntAny("m_FontSize"),
+		NetIntRange("m_FontAlign", 0, 2),
+		NetIntRange("m_MaxTextWidth", 0, 100),		
+	]),
+
+	NetMessage("ModAPI_Sv_GuiButtonMenu", [
```

Comment:
Because it is called exactly the same in vanilla tw. I just took the names like they were, but I can remove it for you.
(Btw, I'm fixing and cleaning up all that stuff you mentioned, locally. It will be pushed soon)